DiscordChatExporter.Cli/Commands/Base/ExportCommandBase.cs

```diff
@@ -24,8 +24,9 @@ public abstract class ExportCommandBase : TokenCommandBase
         [CommandOption("before", Description = "Only include messages sent before this date or message ID.")]
         public Snowflake? Before { get; init; }
 
-        [CommandOption("partition", 'p', Description = "Split output into partitions limited to this number of messages.")]
-        public int? PartitionLimit { get; init; }
+        [CommandOption("partition", 'p', Description = "Split output into partitions limited to this number of messages or a maximum file size (e.g. \"25mb\").",
+            Validators = new[] { typeof(PartitionValidator) })]
+        public string? PartitionLimit { get; init; }
```

Comment:
What I was thinking was to use `IPartitioner` right here, instead of `string`. We can do it either by providing a converter, or by changing the interface to an abstract class and adding `Partitioner.Parse(string)` method.